docs/source/en/perf_train_cpu_many.mdx

```diff
@@ -37,7 +37,14 @@ pip install oneccl_bind_pt=={pytorch_version} -f https://software.intel.com/ipex
 where `{pytorch_version}` should be your PyTorch version, for instance 1.12.0.
 Check more approaches for [oneccl_bind_pt installation](https://github.com/intel/torch-ccl).
 
-### Usage in Trainer
+## Intel® MPI library
+Use this standards-based MPI implementation to deliver flexible, efficient, scalable cluster messaging on Intel® architecture. This component is part of the Intel® oneAPI HPC Toolkit.
+it could be installed from [mpi installation](https://www.intel.com/content/www/us/en/developer/articles/tool/oneapi-standalone-components.html#mpi).
```

Comment:
```suggestion
It can be installed via [MPI](https://www.intel.com/content/www/us/en/developer/articles/tool/oneapi-standalone-components.html#mpi).
```